content/2019/10/01/mozdev.rst

```diff
@@ -1,10 +1,8 @@
 public: yes
 author: miriam
-canonical: https://hacks.mozilla.org/2019/10/video-shorts-from-mozilla-developer/
 headline:
   - tagline: a gift from Mozilla (and OddBird)
-elsewhere: Mozilla Hacks
-tags: [Elsewhere, CSS, 'Mozilla Hacks', Code, Video]
+tags: [CSS, 'Mozilla Hacks', Code, Video]
```

Comment:
This one uses `Mozilla Hacks` while the new post uses `Mozilla Developer` -- should they be the same?

Reply:
possibly. Mozilla Hacks is the blog where we cribbed most of that article (by me in both places). Mozilla Developer is the name of the channel. Maybe use the latter for both now  that we're not linking directly to the hacks article?

Reply:
Changed.